components/webext-storage/src/api.rs

```diff
@@ -189,9 +198,16 @@ fn get_keys(keys: JsonValue) -> Vec<(String, Option<JsonValue>)> {
 
 /// The implementation of `storage[.sync].get()` - on success this always
 /// returns a Json object.
-pub fn get(conn: &Connection, ext_id: &str, keys: JsonValue) -> Result<JsonValue> {
+pub fn get(conn: &mut Connection, ext_id: &str, keys: JsonValue) -> Result<JsonValue> {
+    let mut tx = Transaction::new(conn, TransactionBehavior::Deferred)?;
+    let result = get_in_tx(&mut tx, ext_id, keys)?;
+    tx.commit()?;
```

Comment:
eg, is this the correct pattern to use here? I'd have thought not! But `.rollback()` also seems wrong.

Reply:
What's surprising about this?

Reply:
It was surprising that I'd need to commit/rollback a transaction I know did nothing. It's moot in my new version anyway - get isn't done in a transaction (but get doesn't know that :)